windows/src/buildtools/mkver/Main.pas

```diff
@@ -359,12 +358,9 @@ procedure Run;
   if not Init then
   begin
     UpdateFiles.Free;
-    writeln(#13#10+'Usage: kmver -i '{[-m|-n] }+'[-d] <template> <root>');
+    writeln(#13#10+'Usage: kmver -c <root-version.txt> (-b <version> | -r <version.md>)');
```

Comment:
Can `<version.md>` be removed?

It looks like the parameter isn't even used in l.209 of the Makefile.

And do you want to add `-b` and `-r` flags below?

Reply:
`Makefile` uses both `-b` and `-r` parameters, as shown:

```Make
!ifdef BUILD_NUMBER
    # If CI has defined a version for us, then use that.
    $(MKVER_APP) -c $(ROOT)\src\version.txt -b $(BUILD_NUMBER)
!else
    # Otherwise, we use the major version from resources/VERSION.md and build 9999 for 'debug'
    $(MKVER_APP) -c $(ROOT)\src\version.txt -r $(KEYMAN_ROOT)\resources\VERSION.md
!endif
```

Reply:
> And do you want to add `-b` and `-r` flags below?

I was asking about `writeln`'s below l.363.

Reply:
I see. I have added `-b` and `-r` documentation but I can't remove `<version.md>` -- it is used :)